test/topology/test_topology_ip.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test replacing node in different scenarios
+"""
+import time
+from test.pylib.scylla_cluster import ReplaceConfig
+from test.pylib.manager_client import ManagerClient
+from test.topology.util import wait_for_token_ring_and_group0_consistency
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_replace_different_ip(manager: ManagerClient) -> None:
+    """Replace an existing node with new node using a different IP address"""
```

Comment:
I'm still confused... What "replace an existing node with a new node" mean? Does it mean the new node has the token ranges of the old node but not its data? Or what? Can you relate this to a "nodetool" operation or some familiar administration scenario, for someone who doesn't live an breath these scenarios?

Reply:
https://docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/replace-dead-node.html

The replacing node takes over the old node's token ranges.

If the data belonging to these ranges was replicated to other nodes, it's supposed to get streamed/repaired to the replacing node.